pkg/controller/perconaservermongodb/backup.go

```diff
@@ -280,5 +282,61 @@ func (r *ReconcilePerconaServerMongoDB) updatePITR(cr *api.PerconaServerMongoDB)
 		}
 	}
 
+	val, err = pbm.C.GetConfigVar("pitr.compression")
+	var compression = ""
+	if err != nil {
+		if errors.Is(err, mongo.ErrNoDocuments) {
+			return nil
+		} else if !errors.Is(err, bsoncore.ErrElementNotFound) {
+			return errors.Wrap(err, "get pitr.compression")
+		}
+	} else {
+		compression, ok = val.(string)
+		if !ok {
+			return errors.Wrap(err, "unexpected value of pitr.compression")
+		}
+	}
+
+	if compression != string(cr.Spec.Backup.PITR.Compression) {
+		if string(cr.Spec.Backup.PITR.Compression) == "" {
+			if err := pbm.C.DeleteConfigVar("pitr.compression"); err != nil {
```

Comment:
`pbm.C.DeleteConfigVar()` is not available in pbm main branch yet.
https://github.com/percona/percona-backup-mongodb/pull/629 will bring it

Reply:
Yes, that's what I wrote above 😛

Reply:
could you please make a draft PR if it is not ready next time? I was confused because of it.
P.S. rest of the code changes look good

Reply:
OK.
I wasn't sure if the dependency on the PBM update was a reason for staying as "Draft".
Will do next time.

Reply:
I'd very much like to wrap this PR up but, unfortunately, https://github.com/percona/percona-backup-mongodb/pull/629 hasn't been reviewed yet.
Can someone help get this going? Thanks!